conf/cmi/paragraphs.paragraphs_type.journey_planner.yml

```diff
@@ -0,0 +1,10 @@
+uuid: 90392db0-a1b5-4348-a2b9-c672d59d6f3d
```

Comment:
The paragraph type must be defined in `hook_helfi_paragraph_types()` as well. `modules/custom/helfi_kymp_content/helfi_kymp_content.module` is probably good place for it.

See https://github.com/City-of-Helsinki/drupal-helfi-platform-config/blob/main/documentation/development.md#paragraph-types for documentation.

Reply:
Added to the suggest file.